phenol-cli/src/main/java/org/monarchinitiative/phenol/cli/demo/GoEnrichmentDemo.java

```diff
@@ -95,7 +95,7 @@ public void run() {
         }
         n_sig++;
         double studypercentage=100.0*(double)item.getAnnotatedStudyGenes()/studysize;
-        double poppercentage=(double)item.getAnnotatedPopulationGenes()/popsize;
+        double poppercentage=100.0*(double)item.getAnnotatedPopulationGenes()/popsize;
```

Comment:
im going to be extra nit picky.. the spacing here

Reply:
fixed